packages/charts/src/components/chart-tooltip/chart-tooltip.component.tsx

```diff
@@ -65,6 +71,7 @@ const ChartTooltip: FC<Props> = ({ children, visible, x, y }) => {
             <Tooltip
               mode={theme.tooltip.mode}
               arrowDirection={arrowDirection as Position}
+              arrowTop={typeof arrowTop === 'number' && `${arrowTop}px`}
```

Comment:
Base on `calculateTooltipPosition` the `arrowTop` could be _undefined_ or _number_. Do we need to check typeof arrowTop ?

I believe this code could be replaced with:
```
arrowTop && `${arrowTop}px`;
```